content/doc/book/managing/plugins.adoc

```diff
@@ -225,6 +225,67 @@ The configuration(s) created by the disabled plugin behave as if the plugin
 were uninstalled, insofar that they result in warnings on boot but are
 otherwise ignored.
 
+==== Using the Jenkins CLI
+
+It is also possible to enable and disable plugins using the <<cli#,Jenkins CLI>>.
+Since Jenkins 2.136, administrators may invoke the `enable-plugin` command to 
+re-enable again a collection of plugins which were previously disabled and since
+Jenkins 2.151, the `disable-plugin` command is included, which provides a command
+to disable installed plugins.
+
+The `enable-plugin` command receives as parameter a list of plugins to be enabled.
+If any of the plugins have one or more dependencies, these plugins will be enabled
+as well.
+
+[source]
+----
+java -jar jenkins-cli.jar -s http://localhost:8080/ enable-plugin PLUGIN ... [-restart]
+
+Enables one or more installed plugins transitively.
+
+ PLUGIN   : Enables the plugins with the given short names and their
+            dependencies.
+ -restart : Restart Jenkins after enabling plugins.
+----
+
+The `disable-plugin` command receives as parameter a list of plugins to disable.
+The output will display messages for both successful and failed operations. In 
+case that only the error messages are desired, the execution should add the 
```

Comment:
Maybe something like "the -quiet option can be specified/used"?

Reply:
Yes, sounds better